Gemfile

```diff
@@ -1,7 +1,7 @@
 source "http://rubygems.org"
 
 gem 'resque', '~>1.21'
-gem "ffi-rzmq", "~>1.0"
+gem "ffi-rzmq", "~>2.0"
```

Comment:
Or should it be here because we don't want to force these gems on consumers if they aren't using them? That would make sense.